src/source.ts

```diff
@@ -25,6 +25,22 @@ function issuesFiletypes(): string[] {
   return workspace.getConfiguration().get<string[]>('coc.source.issues.filetypes')
 }
 
+function renderWord(issue: Issue, issueFormat: string): string {
+  return issueFormat.split(/(%i|%o|%r|%b|%t|%c|%a|%u)/).map(part => {
+    switch (part) {
+      case '%i': return issue.id
+      case '%o': return issue.repo.split('/')[0]
+      case '%r': return issue.repo.split('/')[1]
```

Comment:
I've assumed that a `/` is always present. Do you think that is ok?

Reply:
It's ok